examples/music_sfx_pools.rs

```diff
@@ -0,0 +1,355 @@
+//!
+//! Simple setup for a game: general, music and sfx channels settings
+//!
+//! This example set's up the following structure:
+//!
+//! ┌─────┐┌───┐┌───────┐
+//! │Music││Sfx││General│
+//! └┬────┘└┬──┘└┬──────┘
+//! ┌▽──────▽┐   │
+//! │Bus 1   │   │
+//! └┬───────┘   │
+//! ┌▽───────────▽┐
+//! │MainBus      │
+//! └─────────────┘
```

Comment:
```suggestion
//! ```text
//! ┌─────┐┌───┐┌───────┐
//! │Music││Sfx││General│
//! └┬────┘└┬──┘└┬──────┘
//! ┌▽──────▽┐   │
//! │Bus 1   │   │
//! └┬───────┘   │
//! ┌▽───────────▽┐
//! │MainBus      │
//! └─────────────┘
//! ```
```

It will show up as a code block and tell `cargo doc` that it is not, in fact, Rust code.